data/json/npcs/godco/NPC_Aunt_Theresa.json

```diff
@@ -0,0 +1,239 @@
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Theresa_Knit_Something",
+    "dynamic_line": "I suppose I could, in exchange for a few icons.  A scarf is five, a blanket ten, a quilt thirty.",
+    "responses": [
+      {
+        "text": "I'll have a scarf.",
+        "condition": { "u_has_items": { "item": "icon", "count": 5 } },
+        "effect": [
+          { "npc_add_var": "u_want_scarf", "type": "general", "context": "trade", "value": "yes" },
+          { "npc_add_var": "knitting", "type": "timer", "context": "trade", "time": true },
+          { "u_sell_item": "icon", "count": 5 }
+        ],
+        "topic": "TALK_DONE"
+      },
+      {
+        "text": "I'll have a blanket.",
+        "condition": { "u_has_items": { "item": "icon", "count": 10 } },
+        "effect": [
+          { "npc_add_var": "u_want_blanket", "type": "general", "context": "trade", "value": "yes" },
+          { "npc_add_var": "knitting", "type": "timer", "context": "trade", "time": true },
+          { "u_sell_item": "icon", "count": 10 }
+        ],
+        "topic": "TALK_DONE"
+      },
+      {
+        "text": "I'll have a quilt.",
+        "condition": { "u_has_items": { "item": "icon", "count": 30 } },
+        "effect": [
+          { "npc_add_var": "u_want_quilt", "type": "general", "context": "trade", "value": "yes" },
+          { "npc_add_var": "knitting", "type": "timer", "context": "trade", "time": true },
+          { "u_sell_item": "icon", "count": 30 }
+        ],
+        "topic": "TALK_DONE"
+      },
+      { "text": "How long does it take you to make these?", "topic": "TALK_GODCO_Theresa_Knit_Time" },
+      { "text": "I've changed my mind.  How're you feeling?", "topic": "TALK_GODCO_Theresa_2" },
+      { "text": "I've changed my mind, I have to leave.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Theresa_Knit_Time",
+    "dynamic_line": "I can knit a scarf ready by tomorrow, a blanket in two days, and a quilt in a week.",
+    "responses": [ { "text": "Thanks.", "topic": "TALK_GODCO_Theresa_Knit_Something" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Theresa_Knit_Done",
+    "dynamic_line": "I hope you like it <granny_name_g>.  If you want anything else, just come back.",
+    "responses": [ { "text": "Thank you, Theresa.  I'll put it to good use.", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Theresa_Cataclysm",
+    "dynamic_line": "I've grown tired, <granny_name_g>.  I haven't left here since we came, and things outside are getting worse each day.  All my old aches and pains have come back too; I can't see too good, and my muscles are always sore.  I don't like where my daughter is going either; she's ever more fanatical, and taking my dear husband's prophecy to the extreme.  I'm content with my life, though.  I've lived long and prospered by the grace of the Lord.",
```

Comment:
`I don't like where my daughter is going either`  is the daughter different from the niece?

Reply:
No, that's a misprint.